engine/tests/phpunit/integration/Elgg/Database/AnnotationsTableIntegrationTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Elgg\Database;
+
+use Elgg\IntegrationTestCase;
+
+class AnnotationsTableIntegrationTest extends IntegrationTestCase {
+
+	/**
+	 * @var AnnotationsTable
+	 */
+	protected $service;
+
+	/**
+	 * @var \ElggUser
+	 */
+	protected $owner;
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	public function up() {
+		$this->service = _elgg_services()->annotationsTable;
+		$this->owner = $this->createUser();
+		_elgg_services()->session->setLoggedInUser($this->owner);
+	}
+
+	public function testCreateAnnotation() {
+		$entity = $this->createObject();
+
+		$annotation = new \ElggAnnotation();
+		$annotation->owner_guid = $this->owner->guid;
+		$annotation->name = 'foo';
+		$annotation->value = 'bar';
+		
+		$this->assertIsInt($this->service->create($annotation, $entity));
```

Comment:
You didn't check that the correct data was saved to the database.
After the save, load it from the database and verify the data is saved correctly